ccd-definition/CaseEventToFields/WorkAllocationIntegrationFields-EA-nonprod.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "CaseFieldID": "eaCourtLocation",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "FieldShowCondition": "drawDirectionsOrderRequired=\"DONOTSHOW\"",
+    "PageID": "WorkAllocationIntegrationFields",
+    "PageDisplayOrder": 1,
+    "Publish": "Y"
```

Comment:
do these fields need retain hidden value?

Reply:
I don't believe so as we can see the case data being sent in WA pods to trigger the task